src/IconButton/test/IconButtonSpec.tsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
-import { getDOMNode } from '@test/utils';
 import { testStandardProps } from '@test/utils';
 import User from '@rsuite/icons/legacy/User';
 import IconButton from '../IconButton';
+import { render } from '@testing-library/react';
 
 describe('IconButton', () => {
   testStandardProps(<IconButton />);
 
   it('Should output a button', () => {
-    const instance = getDOMNode(<IconButton />);
-    assert.include(instance.className, 'rs-btn-icon');
-    assert.equal(instance.nodeName, 'BUTTON');
+    const { container } = render(<IconButton />);
+
+    expect(container.firstChild).to.have.class('rs-btn-icon');
```

Comment:
According to the priority of Testing Library, we try to use `getByRole` first
https://testing-library.com/docs/queries/about/#priority
```tsx
import {  screen } from '@testing-library/react';

expect(screen.getByRole('button')).to.have.class('rs-btn-icon');
```